### ISSUE TITLE
Failed compilation of pdftoipe (Poppler 22.12.0)

### ISSUE THREAD

**Issue by olin256** [otfried/ipe-tools#62]
Both with and without #48 I'm getting
```
xmloutputdev.cpp:156:21: error: no matching function for call to 'GfxState::getLineDash(double**, int*, double*)'
  156 |   state->getLineDash(&dash, &length, &start);
      |   ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
```

Even #55 did not help and produced
```
pdftoipe.cpp:103:53: error: no matching function for call to 'PDFDoc::PDFDoc(GooString*&, GooString*&, GooString*&)'
  103 |   PDFDoc *doc = new PDFDoc(fileName, ownerPW, userPW);
      |
```

**Comment by occludedpixel** [comment]
@olin256 did you apply both #48 and #55 **together**? The first error should be fixed by #55. The second error should be fixed by #48.

**Comment by otfried** [comment]
Ubuntu 22.04 still ships with libpoppler 22.02, so I cannot just apply those PRs without breaking it for lots of people.

Perhaps we could merge the two PR.  Or someone could figure out the necessary #if's to make it compile with both the old and the new version of the library.
